src/main/java/com/github/hokkaydo/eplbot/module/code/CodeCommand.java

```diff
@@ -0,0 +1,220 @@
+package com.github.hokkaydo.eplbot.module.code;
+import com.github.hokkaydo.eplbot.Config;
+import com.github.hokkaydo.eplbot.Strings;
+import com.github.hokkaydo.eplbot.command.Command;
+import com.github.hokkaydo.eplbot.command.CommandContext;
+
+import net.dv8tion.jda.api.entities.Guild;
+import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
+import net.dv8tion.jda.api.events.interaction.ModalInteractionEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+import net.dv8tion.jda.api.interactions.InteractionType;
+import net.dv8tion.jda.api.interactions.commands.OptionType;
+import net.dv8tion.jda.api.interactions.commands.build.OptionData;
+import net.dv8tion.jda.api.interactions.components.text.TextInput;
+import net.dv8tion.jda.api.interactions.components.text.TextInputStyle;
+import net.dv8tion.jda.api.interactions.modals.Modal;
+import net.dv8tion.jda.api.interactions.modals.ModalMapping;
+import net.dv8tion.jda.api.utils.FileUpload;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Optional;
+import java.util.function.Supplier;
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.stream.Collectors;
+import java.util.List;
+import java.util.Objects;
+import java.time.Instant;
+import java.util.Map;
+
+
+public class CodeCommand extends ListenerAdapter implements Command{
+    private static final String TEMP_DIR = System.getProperty("user.dir")+File.separator+"src"+File.separator+"temp";
+    private final Map<String, Runner> RUNNER_MAP;
+    public CodeCommand() {
+        RUNNER_MAP = Map.of(
```

Comment:
Do you need to recreate new runners instances for each CodeCommand instance (one instance is for one server)